src/components/people.tsx

```diff
@@ -0,0 +1,177 @@
+                  >
+                    <thead>
+                      <tr>
+                        <th>
+                          <span className="is-flex is-flex-wrap-nowrap">
+                            Name
+                            <Link
+                              to="/people?sort=name"
+                              onClick={e => sortPeopleByField('name', e)}
+                            >
+                              <span className="icon">
+                                <i className={getSortIcon('name')} />
+                              </span>
+                            </Link>
+                          </span>
+                        </th>
+                        <th>
+                          <span className="is-flex is-flex-wrap-nowrap">
+                            Sex
+                            <Link
+                              to="/people?sort=sex"
+                              onClick={e => sortPeopleByField('sex', e)}
+                            >
+                              <span className="icon">
+                                <i className={getSortIcon('sex')} />
+                              </span>
+                            </Link>
+                          </span>
+                        </th>
+                        <th>
+                          <span className="is-flex is-flex-wrap-nowrap">
+                            Born
+                            <Link
+                              to="/people?sort=born"
+                              onClick={e => sortPeopleByField('born', e)}
+                            >
+                              <span className="icon">
+                                <i className={getSortIcon('born')} />
+                              </span>
+                            </Link>
+                          </span>
+                        </th>
+                        <th>
+                          <span className="is-flex is-flex-wrap-nowrap">
+                            Died
+                            <Link
+                              to="/people?sort=died"
+                              onClick={e => sortPeopleByField('died', e)}
+                            >
+                              <span className="icon">
+                                <i className={getSortIcon('died')} />
+                              </span>
+                            </Link>
+                          </span>
+                        </th>
+                        <th>Mother</th>
+                        <th>Father</th>
+                      </tr>
+                    </thead>
+
+                    <tbody>
+                      {person.map((pers: PersonType) => (
+                        <Person key={pers.slug} pers={pers} />
+                      ))}
+                    </tbody>
+                  </table>
+                </>
+              )}
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  );
```

Comment:
The return statement of this component is quite large and includes a lot of JSX. Consider breaking it down into smaller, reusable components. This would make your code cleaner and easier to understand.